vmlinux_to_elf/main.py

```diff
@@ -16,6 +16,8 @@
     from vmlinux_to_elf.architecture_detecter import ArchitectureGuessError
 
 if __name__ == '__main__':
+
+    logging.basicConfig(level=logging.INFO, format='%(message)s')
```

Comment:
It seems that you forgot to add the logging level configuration to all entry points of the program (including `kallsyms_finder.py`), which causes no output to be printed when using these.